rust-src/concordium_base/src/encrypted_transfers/proofs/dlogaggequal.rs

```diff
@@ -132,12 +134,21 @@ impl<C: Curve> SigmaProtocol for DlogAndAggregateDlogsEqual<C> {
         }
         Some((dlog_point, agg_points))
     }
+
+    #[cfg(test)]
+    fn with_valid_data<R: rand::Rng>(
```

Comment:
What is this for?

Reply:
This is part of the trait to be used in tests of sigma protocols.

It's not the best solution, but I am not introducing new stuff here, just making the necessary changes for migration.